test/workflows/components/workflows.libsonnet

```diff
@@ -356,6 +360,12 @@
               "--context=dir://" + srcDir + "/python",
               "--destination=" + "527798164940.dkr.ecr.us-west-2.amazonaws.com/kfserving/aix-explainer:$(PULL_BASE_SHA)",
             ]),  // build-aix-explainer
+            $.parts(namespace, name, overrides).e2e(prow_env, bucket).buildTemplate("build-art-explainer", kanikoExecutorImage, [
+              "/kaniko/executor",
+              "--dockerfile=" + srcDir + "/python/artexplainer.Dockerfile",
+              "--context=dir://" + srcDir + "/python",
+              "--destination=" + "527798164940.dkr.ecr.us-west-2.amazonaws.com/kfserving/art-explainer:$(PULL_BASE_SHA)",
```

Comment:
is this where the image will pushed once built?

Reply:
that's the ci test image, the release image is under `.github/workflows/artexplainer-docker-publish.yml`